### ISSUE TITLE
Provide CEPC preamble for Chairs

### ISSUE THREAD

**Issue by msporny** [w3c/PWETF#152]
We should provide a CEPC preamble for Chairs to read to their groups every two months or so, to remind the group of the CEPC. The preamble should spend a small part of the time noting the CEPC exists, where to find it, and that the meeting is covered by it... but should spend the majority of the time with the most common examples of bad behavior... 

For example: If you find yourself saying "well, actually" a lot you might be nitpicking and not adding much to the conversation. If you think someone else in the group is acting in bad faith, try assuming that they're acting in good faith and answering them if the latter was true... and so on. Most importantly, it should highlight that it is everyone's job to intervene in group CEPC issues, not just the Chairs.

**Comment by msporny** [comment]
Hey @vsnt, @wyc, @kimdhamilton, 

There was a really great set of examples that @vsnt and @wyc read out loud on one of the W3C CCG meetings... do you have the text from that? Remember which meeting that was? It was the best "CEPC Cheatsheet" I've heard so far...

**Comment by TzviyaSiegman** [comment]
Thanks! This is great resource

**Comment by wareid** [comment]
I would love to have a "CEPC Gentle Reminder" or something to read out or point to on the regular. I tried to do something like this in my first WG meeting a few weeks ago and found myself kind of stumbling through it.

**Comment by msporny** [comment]
> I was borrowing the Recurse Center's social rules

Yep, that was it... something pithy that can be said quickly and doesn't lose people's attention. Focusing on a different example every time you do it can sometimes help keep folks attention. If it turns into reading boilerplate, folks will tune out.

I wondering if starting every preamble with an example would be useful:

Tzviya: Welcome to the Foo Working Group, remember that this meeting is covered by the W3C Code of Conduct.

Jerkface1: Well, actually, it's called the Code of Ethics and Professional Conduct, Tzviya, I'm surprised you didn't know that.

Tzviya: Thanks for highlighting two examples of CEPC Unacceptable Behavior, Jerkface1... they were well-actuallys and feigned surprise, &lt;explanation here>

...

Tzviya: Remember that we're all here to make the Web better and it's not only the Chairs responsibility, but all of our responsibility to make sure everyone is being treated in an ethical and professional manner.

**Comment by vsnt** [comment]
Yes, and we also like to remind people when commenting during the discussion to keep it to one point and be pithy because we can all (I am guilty!) ramble. 

Having these gentle reminders for us all really helps have a productive and pleasant conversation. Also being kind when reminding someone and realizing that we may all make mistakes from time to time (compassion).

**Comment by npdoty** [comment]
+1 to preambles and short, pithy summaries that can work as reminders.

When I am acting as chair, I remind participants at the beginning of every meeting, and try to call attention to a different section of the CEPC each time, as an opportunity to get people to think about the different points (which would be too long to get into without a longer agenda item) and to avoid the scenario where it just becomes identical boilerplate that is typically ignored.

To that end, I would prefer not to have a single standard boilerplate preamble, but a selection of short text suggestions.

**Comment by cwilso** [comment]
Also +1.  I've seen (in other venues) mention of their CoC that made me feel like "they're mentioning it, which is good, but mentioning the CoC in a way that makes it seem... unimportant?"  so it would be good to have some suggestions of how to do it in a crisp, pragmatic way that communicates the value.

**Comment by TzviyaSiegman** [comment]
Thanks for raising this again @npdoty . This suggestion got buried with some of the work that we've done. I will get to work on drafting something.

**Comment by TzviyaSiegman** [comment]
fixed with ](https://github.com/w3c/guide/pull/430)
